src/api/procedures/setAssetRequirements.ts

```diff
@@ -103,7 +100,7 @@ export function getAuthorization(
     permissions: {
       transactions: [
         TxTags.complianceManager.ResetAssetCompliance,
```

Comment:
Should we check if it is going to be either Reset or Replace for authorization purposes?

Reply:
Yeah, makes sense